dot/peerset/peerstate.go

```diff
@@ -171,6 +175,9 @@ func (ps *PeersState) peerStatus(set int, peerID peer.ID) string {
 
 // peers return the list of all the peers we know of.
 func (ps *PeersState) peers() []peer.ID {
+	ps.mu.RLock()
```

Comment:
perhaps name `mu` with a name so we know what it's protecting? I guess `peerStateMutex`??